### ISSUE TITLE
[Accessibility] ActivityUpdate: After activating the 'Done' button voice over focus is landing on the 'Set due date' button.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6288]
### Target Platforms

iOS

### SDK Version

App Version: Version 1.0 (2.3.1-beta.20210603.1)

### Application Name

Visualizer

### Problem Description

[33754614](https://microsoft.visualstudio.com/OS/_workitems/edit/33754614)

After activating the 'Done' button under numeric pane, Voice Over focus is landing on the 'Set due date' button. After activating the 'Done' button voice over focus should be land on the 'OK' button below comment pane or comment pane.

### Card JSON

[ActivityUpdate.json](https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.0/Scenarios/ActivityUpdate.json)

**Comment by vagpt** [comment]
working fine on the latest build hence closing this issue.
